### PR TITLE
feat: Send OS Version and Device Model to config API

### PR DESCRIPTION
**Pull request by corycaywood**
# Description 
Sends `Build.VERSION.RELEASE` for the `osVersion` and `Build.MODEL` for the `device`.

## Links
APTT-323

# Checklist
- [x] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [x] I wrote/updated tests for new/changed code
- [x] I ran `./gradlew check` without errors

### CODE REVIEW

File: Runtime/src/test/java/com/rakuten/tech/mobile/perf/runtime/internal/ConfigurationRequestSpec.java [hunk 1]
```diff
@@ -33,27 +33,88 @@
...
+        .contains("/version/testVersion/");
+  }
+
+  @Test public void shouldBuildUrlWithSdkVersion() {
+    builder.setSdkVersion("testSdkVersion");
+
+    ConfigurationRequest request = new ConfigurationRequest(
+        null, "", builder.build(), null, null);
+    assertThat(request.getUrl())
+        .contains("?sdk=testSdkVersion");
```
Comment (NemoOudeis) [inline Runtime/src/test/java/com/rakuten/tech/mobile/perf/runtime/internal/ConfigurationRequestSpec.java]: the fact that `sdk` is the first parameter is an implementation detail, isn't it?

File: Runtime/src/test/java/com/rakuten/tech/mobile/perf/runtime/internal/ConfigurationRequestSpec.java [hunk 2]
```diff
@@ -33,27 +33,88 @@
...
+  }
+
+  @Test public void shouldBuildUrlWithCountryCode() {
+    builder.setCountryCode("testCountryCode");
+
+    ConfigurationRequest request = new ConfigurationRequest(
+        null, "", builder.build(), null, null);
+    assertThat(request.getUrl())
+        .contains("&country=testCountryCode");
```
Comment (NemoOudeis) [inline Runtime/src/test/java/com/rakuten/tech/mobile/perf/runtime/internal/ConfigurationRequestSpec.java]: the fact that `country` is the first parameter is an implementation detail, isn't it?

File: Runtime/src/test/java/com/rakuten/tech/mobile/perf/runtime/internal/ConfigurationRequestSpec.java [hunk 3]
```diff
@@ -33,27 +33,88 @@
...
+  }
+
+  @Test public void shouldBuildUrlWithOsVersion() {
+    builder.setOsVersion("testOsVersion");
+
+    ConfigurationRequest request = new ConfigurationRequest(
+        null, "", builder.build(), null, null);
+    assertThat(request.getUrl())
+        .contains("&osVersion=testOsVersion");
```
Comment (NemoOudeis) [inline Runtime/src/test/java/com/rakuten/tech/mobile/perf/runtime/internal/ConfigurationRequestSpec.java]: same here

File: Runtime/src/test/java/com/rakuten/tech/mobile/perf/runtime/internal/ConfigurationRequestSpec.java [hunk 4]
```diff
@@ -33,27 +33,88 @@
...
+  }
+
+  @Test public void shouldBuildUrlWithDevice() {
+    builder.setDevice("test device name");
+
+    ConfigurationRequest request = new ConfigurationRequest(
+        null, "", builder.build(), null, null);
+    assertThat(request.getUrl())
+        .contains("&device=test%20device%20name");
```
Comment (NemoOudeis) [inline Runtime/src/test/java/com/rakuten/tech/mobile/perf/runtime/internal/ConfigurationRequestSpec.java]: and here

**Review by NemoOudeis — CHANGES_REQUESTED** [review]

File: Runtime/src/test/java/com/rakuten/tech/mobile/perf/runtime/internal/ConfigurationRequestSpec.java (hunk 1 above)
Comment (corycaywood) [reply Runtime/src/test/java/com/rakuten/tech/mobile/perf/runtime/internal/ConfigurationRequestSpec.java]: Yep, I will change these.

**Review by NemoOudeis — APPROVED** [review]

**Pull request merged**